nais/dev-gcp.json

```diff
@@ -7,6 +7,9 @@
   "kafka-pool" : "nav-dev",
   "minReplicas": "1",
   "maxReplicas": "2",
+  "ingresses": [
+    "https://omsorgspengerutbetalingsoknad-prosessering.dev.intern.nav.no"
```

Comment:
Er den noe grunn til å legge på ingress på denne tjenesten?

Reply:
Bara for standardisering, alla andra prosesseringsapplikasjonerna har det & naiserator.yaml har i tillægg ingresses men den blir ikke brukt. Copy paste feil? :)

Reply:
Ok, men tviler på at det er nødvendig med ingresses på en ren kafka-consumer. 
Ingresses i naiserator.yaml gjenstår sikkert fra før ingressen ble slettet.

Vi har plan om å fjerne ingressene i de andre prosesseringstjenestene våres også.

Reply:
Aha, ja då gør vi det åt andra hållet ist. :)